### PR TITLE
Kill orgs

### PR DESCRIPTION
**Pull request by dannyroberts**

### CODE REVIEW

File: corehq/apps/orgs/forms.py [hunk 1]
```diff
@@ -2,11 +2,13 @@
 from django.core.validators import validate_email
 from corehq.apps.domain.models import Domain
 import re
-from corehq.apps.domain.utils import new_domain_re, website_re
+from corehq.apps.domain.utils import website_re
 from corehq.apps.orgs.models import Organization, Team, OrgRequest
-from corehq.apps.registration.forms import OrganizationRegistrationForm
 from corehq.apps.users.models import CouchUser
 
+new_org_re = r"(?:[a-z0-9]+\-)*[a-zA-Z0-9]+" # lowercase and uppercase letters, numbers, and '-' (at most one between "words")
```
Comment (dimagimon) [inline corehq/apps/orgs/forms.py]: E261 at least two spaces before inline comment

Comment (dimagimon) [inline corehq/apps/orgs/forms.py]: E501 line too long (126 > 115 characters)

File: corehq/apps/orgs/forms.py [hunk 2]
```diff
@@ -97,6 +99,58 @@ def clean_team(self):
...
+    org_name = forms.CharField(label='Organization ID:', max_length=25, help_text='e.g. - dimagi')
+    email = forms.CharField(label='Organization Email:', max_length=35, required=False)
+    url = forms.CharField(label='Organization Homepage:', max_length=35, required=False)
+    location = forms.CharField(label='Organization Location:', max_length=25, required=False)
+    # logo = forms.ImageField(label='Organization Logo:', required=False)
+
+    def clean_org_name(self):
+        data = self.cleaned_data['org_name'].strip().lower()
+        if not re.match("^%s$" % new_org_re, data):
+            raise forms.ValidationError('Only lowercase letters and numbers allowed. Single hyphens may be used to separate words.')
```
Comment (dimagimon) [inline corehq/apps/orgs/forms.py]: E501 line too long (132 > 115 characters)

**Comment by snopoke** [comment]
:trophy:

**Comment by czue** [comment]
woohoo!

**Comment by orangejenny** [comment]
glorious

**Comment by benrudolph** [comment]
big D coming in hot!

**Comment by dannyroberts** [comment]
@czue was this the test that https://github.com/dimagi/commcare-hq/pull/8730 was made to fix? Re-running failed job.
```
======================================================================
FAIL: testDefaultValue (corehq.apps.reports.tests.test_scheduled_reports.ScheduledReportTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/reports/tests/test_scheduled_reports.py", line 56, in testDefaultValue
    self._check('daily', None, 1)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/reports/tests/test_scheduled_reports.py", line 50, in _check
    self.assertEqual(count, len(list(get_scheduled_reports(period, as_of))))
AssertionError: 1 != 0
----------------------------------------------------------------------
```

**Comment by czue** [comment]
yeah, I think there's another failure mode of that test.

**Comment by czue** [comment]
https://github.com/dimagi/commcare-hq/pull/8749

**Comment by biyeun** [comment]
KILL O_O

**Comment by czue** [comment]
merged master - seemed easy, but fyi in case i messed anything up

**Comment by dannyroberts** [comment]
I just threw this up on staging, and am leaving for a week. If there aren't any issues on staging, I was planning to just deploy this when I get back. @czue @amsagoff any other ideas or concerns?

**Comment by amsagoff** [comment]
none. DIE ORGS DIE

**Comment by czue** [comment]
image

**Comment by dannyroberts** [comment]
I think this is g2g (qa passed just means it's been on staging without issues)

**Comment by czue** [comment]
:boom: :rage1: :bomb:

**Comment by dannyroberts** [comment]
:goberserk:

**Comment by benrudolph** [comment]
:godmode: :goberserk:

**Pull request merged**